<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View_PostWeibo</class>
 <widget class="QMainWindow" name="View_PostWeibo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonCancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 76, 76);
color: rgb(255, 255, 255);
border:2px groove gray;
border-radius:5px;
padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPosition">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>105</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/image/composer-poi@2x copy.png</normaloff>:/image/image/composer-poi@2x copy.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEmotions">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>105</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/image/composer-emoji@2x copy.png</normaloff>:/image/image/composer-emoji@2x copy.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonImage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>105</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/image/rounded-icon-plus@2x copy.png</normaloff>:/image/image/rounded-icon-plus@2x copy.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>105</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 76, 76);
color: rgb(255, 255, 255);
border:2px groove gray;border-radius:5px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_avatar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="newWeiboEditor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>250</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(51, 51, 51);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_counter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(204, 204, 204, 191);
</string>
    </property>
    <property name="text">
     <string>140</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>102</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_image</zorder>
   <zorder>label_counter</zorder>
   <zorder>label_bar</zorder>
   <zorder>buttonCancel</zorder>
   <zorder>buttonPosition</zorder>
   <zorder>buttonEmotions</zorder>
   <zorder>buttonImage</zorder>
   <zorder>buttonSend</zorder>
   <zorder>label_avatar</zorder>
   <zorder>newWeiboEditor</zorder>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
